### PR TITLE
[smoke-test][genesis] enabling configuring genesis onchain configs in test

### PR DESCRIPTION
**Pull request by igor-aptos**
### Test Plan

Used in #4986 

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/5527)

### CODE REVIEW

**Review by msmouse — APPROVED** [review]

**Comment by bchocho** [comment]
@igor-aptos does this need a rebase after landing #4986?

**Comment by bchocho** [comment]
Oh. I saw it was merged and thought it was merged on main

File: testsuite/smoke-test/src/aptos_cli/validator.rs [hunk 1]
```diff
@@ -76,6 +81,269 @@ async fn test_show_validator_set() {
...
+        .nth(2)
+        .map(|v| (v.peer_id(), v.rest_client()))
+        .unwrap();
+
+    rest_client_off
+        .set_failpoint("consensus::send::any".to_string(), "100%return".to_string())
+        .await
+        .unwrap();
+
+    // tokio::time::sleep(Duration::from_secs(20)).await;
```
Comment (bchocho) [inline testsuite/smoke-test/src/aptos_cli/validator.rs:105]: Stray comment

**Review by bchocho — APPROVED** [review]
LGTM

**Comment by igor-aptos** [comment]
Argh, seems like with last update I've added the commit from #4986 here. let me remove it from here, since @msmouse didn't approve that part, and @bchocho you can approve it there, as you already have reviewed it

**Pull request merged**